client/blocks/nav-unification-quick-switch-modal/index.jsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Guide } from '@wordpress/components';
+import { Title } from '@automattic/onboarding';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getPreference, hasReceivedRemotePreferences } from 'calypso/state/preferences/selectors';
+import { getCurrentUserId } from 'calypso/state/current-user/selectors';
+
+/**
+ * Image dependencies
+ */
+import featureImage from 'calypso/assets/images/nav-unification-announcement/quick-switch-feature.png';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const Page = ( { heading, content, image } ) => {
+	return (
+		<div className="nav-unification-quick-switch-modal__page">
+			<div className="nav-unification-quick-switch-modal__text">
+				<div className="nav-unification-quick-switch-modal__heading">
+					<Title tagName="h2">{ heading }</Title>
+				</div>
+				<div className="nav-unification-quick-switch-modal__description">
+					<p>{ content }</p>
+				</div>
+			</div>
+			<div className="nav-unification-quick-switch-modal__visual">{ image }</div>
+		</div>
+	);
+};
+
+const Modal = () => {
+	const dispatch = useDispatch();
+	const userId = useSelector( ( state ) => getCurrentUserId( state ) );
+	const hasPreferences = useSelector( hasReceivedRemotePreferences );
+	const dismissPreference = `nav-unification-quick-switch-modal-${ userId }`;
+	const isDismissed = useSelector( ( state ) => getPreference( state, dismissPreference ) );
+	const translate = useTranslate();
+
+	if ( ! hasPreferences || isDismissed ) {
+		return null;
+	}
+
+	const handleDismiss = () => {
+		dispatch( savePreference( dismissPreference, 1 ) );
+	};
+
+	return (
+		<Guide
+			className="nav-unification-quick-switch-modal"
+			onFinish={ handleDismiss }
+			pages={ [
+				{
+					content: (
+						<Page
+							image={
+								<picture
+									className="nav-unification-quick-switch-modal__picture nav-unification-quick-switch-modal__picture--bottom-left"
```

Comment:
Since we only need one picture, can we avoid using the `nav-unification-quick-switch-modal__picture--bottom-left` class and simplify a bit more the styles?

Reply:
I don't think there's much harm in keeping those styles separate as a modifier. It's clear that those properties are responsible for the images positioning.